TLCGen.PluggedInItems/TLCGen.Generators.CCOL/CodeGeneration/Sources/CCOLGeneratorGenerateRegC.cs

```diff
@@ -610,14 +618,35 @@ private string GenerateRegCFileVerwerking(ControllerModel controller)
         private string GenerateRegCDetectieStoring(ControllerModel controller)
         {
             var sb = new StringBuilder();
+            var storingsopvang = false;
```

Comment:
idem als bij fileverwerking: vanwaar deze check? de #defines had ik al toegevoegd, maar zonder check. ze zitten sowieso ook niet in de weg, dus ik stel voor de check weg te laten

Reply:
Zelfde onderbouwing als bij File ingrepen ;-)
Ook hier alleen kwestie van smaak en geen halszaak.,